heat_infoblox/resources/grid_member.py

```diff
@@ -178,13 +186,51 @@ class GridMember(resource.Resource):
                     _('If true, enable DNS on this member.'),
                     default=False
                 ),
-            })
+            }),
+        HA_PAIR: properties.Schema(
+            properties.Schema.BOOLEAN,
+            _('Count of hardware nodes. Allowed values 1 and 2(for HA pair).'),
```

Comment:
Shouldn't this be true or false, not 1 or 2. Seems like the description is wrong.